### PR TITLE
(wix) update README.md to explain how to test multilingual MSI in a 

### PR DESCRIPTION
**Pull request by sblaisot**

### CODE REVIEW

**Comment by sblaisot** [comment]
I lost 2 hours finding how to test it, so adding it to README.md to save time for the next one who will need to test multilingual MSI

File: build/wix/README.md [hunk 1]
```diff
@@ -12,4 +12,11 @@ For example, to build 64 bits package:
...
-.\build.bat 32
\ No newline at end of file
+.\build.bat 32
+
+To test your multilingual package
+=================================
+
+`msiexec /i mixxx-full-name.msi ProductLanguage=xxxx`
+
+where xxxx is a language code taken from http://www.science.co.il/Language/Locale-codes.asp and available as a translation in build/wix/Localization
```
Comment (daschuer) [inline build/wix/README.md]: The language codes column on http://www.science.co.il/Language/Locale-codes.asp have two chars. 
The build/wix/Localization files uses the rfc5646 notation. 
xxxx does not match any of them.

Comment (sblaisot) [inline build/wix/README.md]: xxxx should be taken from the "LCID decimal" column.
This is 1033 for american english, 1031 for German and 1036 for French and so on.

**Comment by daschuer** [comment]
Thank you! LGTM

**Pull request merged**